flexget/plugins/api_tvmaze.py

```diff
@@ -0,0 +1,402 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+from datetime import datetime
+
+from dateutil import parser
```

Comment:
Maybe we should `from dateutil import parser as date_parser` so that it's obvious when we use it it's a date parser.